airbyte-integrations/connectors/source-mailchimp/source_mailchimp/schemas/Email_activity.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
are you sure that we can find Email_activity schema?

Reply:
also your schema is wrong, here we should have email activity record and schema of endpoint response. We should have schema for the following object
```json
{
  "campaign_id": "string",
  "list_id": "string",
  "list_is_active": true,
  "email_id": "string",
  "email_address": "string",
  "action": "string",
  "type": "string",
  "timestamp": "2019-08-24T14:15:22Z",
  "url": "string",
  "ip": "string"
}
```